apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClient.cs

```diff
@@ -192,26 +211,38 @@ private static BigQueryClient CreateImpl(string projectId, GoogleCredential scop
 
         /// <summary>
         /// Creates a <see cref="JobReference"/> from the given job ID,
-        /// using this client's project ID.
+        /// using this client's project ID and default location.
         /// </summary>
         /// <param name="jobId">The job ID. Must not be null.</param>
         /// <returns>A <see cref="JobReference"/> representing the requested job.</returns>
         public JobReference GetJobReference(string jobId) =>
             GetJobReference(ProjectId, jobId);
 
         /// <summary>
-        /// Creates a <see cref="JobReference"/> from the given project ID and job ID.
+        /// Creates a <see cref="JobReference"/> from the given project ID and job ID, using
+        /// this client's default location.
         /// </summary>
         /// <param name="projectId">The project ID. Must not be null.</param>
         /// <param name="jobId">The job ID. Must not be null.</param>
         /// <returns>A <see cref="JobReference"/> representing the requested job.</returns>
         public JobReference GetJobReference(string projectId, string jobId) =>
+            GetJobReference(projectId, jobId, DefaultLocation);
+
+        /// <summary>
+        /// Creates a <see cref="JobReference"/> from the given project ID, job ID and location.
+        /// </summary>
+        /// <param name="projectId">The project ID. Must not be null.</param>
+        /// <param name="jobId">The job ID. Must not be null.</param>
+        /// <param name="location">The location. May be null.</param>
+        /// <returns>A <see cref="JobReference"/> representing the requested job.</returns>
+        public JobReference GetJobReference(string projectId, string jobId, string location) =>
```

Comment:
Since location can be null, why not a default parameter in the other overload? Is this to avoid binary breaking changes?

Reply:
Yes, exactly that.